components/SettingsControls/src/SettingsCard/SettingsCard.cs

```diff
@@ -217,8 +255,29 @@ private void OnHeaderChanged()
                 ? Visibility.Visible
                 : Visibility.Collapsed;
         }
+       
     }
 
+    private void OnContentChanged(DependencyObject sender, DependencyProperty dp)
```

Comment:
Is there a reason we need this empty method?

Reply:
I got it